src/components/calcite-loader/calcite-loader.scss

```diff
@@ -45,6 +45,10 @@ $loader-circumference: ($loader-scale - (2 * $stroke-width)) * 3.14159;
   @apply py-0;
 }
 
+:host {
+  @apply hidden;
```

Comment:
loader should be hidden by default and displayed when active.

Reply:
Sort of a separate discussion, but maybe we should use `hidden` for this? Looks like it's only used for visibility. cc @macandcheese

Reply:
Its a separate discussion but by default the loader is not shown until it has an active attribute/property. We can evaluate changing this so that the active property is not necessary and just remove it.

Reply:
https://github.com/Esri/calcite-components/issues/3821